src/Test/CacheAssertions.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of the FOSHttpCache package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\HttpCache\Test;
+
+use FOS\HttpCache\Test\PHPUnit\IsCacheHitConstraint;
+use FOS\HttpCache\Test\PHPUnit\IsCacheMissConstraint;
+use Psr\Http\Message\ResponseInterface;
+
+/**
+ * Provides cache hit/miss assertions to PHPUnit tests
+ */
+trait CacheAssertions
```

Comment:
yeah, old discussion. the same goes for `Interface` suffixes. i am fine with this.